builder/internals_windows.go

```diff
@@ -0,0 +1,7 @@
+// +build windows
+
+package builder
+
+func fixPermissions(source, destination string, uid, gid int, destExisted bool) error {
+	// chown is not supported on Windows
```

Comment:
I'm surprised it compiles without a return.
You might want to return `ErrNotSupported` or something like this.